tests/postgres_tests/test_operations.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+
+from tests.migrations.test_base import OperationTestBase
+
+from django.db import connection, models
+from django.db.utils import NotSupportedError
+from django.test import modify_settings
+
+try:
+    from django.contrib.postgres.operations import AddIndexConcurrently, RemoveIndexConcurrently
+    from django.contrib.postgres.indexes import BrinIndex
+except ImportError:
+    pass
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'AddIndexConcurrently is PostgreSQL-specific')
+@modify_settings(INSTALLED_APPS={'append': 'migrations'})
+class AddIndexConcurrentlyTests(OperationTestBase):
+    def test_add_index_concurrently_requires_name(self):
```

Comment:
I don't think the behavior inherited from the superclass needs to be tested again.